datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/codeowners.py

```diff
@@ -18,6 +18,7 @@
     'bonsai',
     'buddy',
     'concourse_ci',
+    'f5-distributed-cloud',
```

Comment:
I only have a question (though it's more closely related to the PR in extras): should underscores be preferred to dashes here, or it doesn't matter? I can only find one existing integration across extras and core that uses dashes.

Reply:
That's a good question. We also only have two integrations in marketplace that use dashes. I think it's ok since the validation allows it but it might be better to use underscore for consistency